softvis3d-frontend/src/style/components/topbar.scss

```diff
@@ -47,4 +50,45 @@ $content-height: 32px;
             @include softvis-button;
         }
     }
+
```

Comment:
@Ungolianth Please take a deeper look at this. I'm not really good with css. It looks good, but I'm sure this is not optimal here.